src/components/Badge.tsx

```diff
@@ -0,0 +1,18 @@
+import ConditionalRender from "components/ConditionalRender"
+
+interface BadgeProps {
+  isRendered: boolean
+  className?: string
+  colour: string
+  label: string
+}
+
+const Badge = ({ isRendered, className, colour, label }: BadgeProps) => {
```

Comment:
This is probably quite a nice pattern in general for things that need showing and hiding based on roles etc